src/apps/rDSN.monitor/app_package/static/view/head.html

```diff
@@ -17,23 +17,36 @@
 <script>
```

Comment:
Remember that we have several views just for task latency? This is the head part for them, including functions like call link graph and task jump and etc.